CHANGELOG.md

```diff
@@ -2,16 +2,22 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.12.3] - 2024-09-30
```

Comment:
why would this bump a version? It's just docs

Reply:
removed